src/SFA.DAS.ApplyService.Application/Apply/Roatp/RoatpPreambleQuestionBuilder.cs

```diff
@@ -123,28 +123,88 @@ public static class YourOrganisation
 
         public static class FinancialEvidence
         {
-            public static int WhatYouWillNeed = 1;
-            public static int YourOrganisationsFinancialEvidence = 2;
+            public const int WhatYouWillNeed = 1;
+            public const int YourOrganisationsFinancialEvidence = 2;
         }
 
         public static class CriminalComplianceChecks
         {
-            public static int WhatYouWillNeed = 1;
-            public static int ChecksOnYourOrganisation = 2;
-            public static int WhatYouWillNeed_CheckOnWhosInControl = 3;
-            public static int CheckOnWhosInControl = 4;
+            public const int WhatYouWillNeed = 1;
+            public const int ChecksOnYourOrganisation = 2;
+            public const int WhatYouWillNeed_CheckOnWhosInControl = 3;
+            public const int CheckOnWhosInControl = 4;
+        }
+
+        public static class ProtectingYourApprentices
+        {
+            public const int WhatYouWillNeed = 1;
+        }
+
+        public static class ReadinessToEngage
+        {
+            public const int WhatYouWillNeed = 1;
+        }
+
+        public static class PlanningApprenticeshipTraining
+        {
+            public const int WhatYouWillNeed = 1;
         }
 
         public static class DeliveringApprenticeshipTraining
         {
-            public static int ManagementHierarchy = 3;
+            public const int WhatYouWillNeed = 1;
+            public const int ManagementHierarchy = 3;
+        }
+
+        public static class EvaluatingApprenticeshipTraining
+        {
+            public const int WhatYouWillNeed = 1;
         }
+
         public static class Finish
         {
-            public static int ApplicationPermissionsAndChecks = 1;
-            public static int CommercialInConfidenceInformation = 2;
-            public static int TermsAndConditions = 3;
-            public static int SubmitApplication = 4;
+            public const int ApplicationPermissionsAndChecks = 1;
+            public const int CommercialInConfidenceInformation = 2;
+            public const int TermsAndConditions = 3;
+            public const int SubmitApplication = 4;
+        }
+
+        public static List<int> GetWhatYouWillNeedSectionsForSequence(int sequenceId)
```

Comment:
This probably belongs in a separate class / file. Wondered if it would be better modelled in configuration?